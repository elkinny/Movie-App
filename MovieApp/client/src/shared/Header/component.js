import React from 'react';

import { StyledHeader, StyledTitle } from './styled.component.js';

const Header = () => {
  return (
    <StyledHeader>
      <StyledTitle>Movie app</StyledTitle>
    </StyledHeader>
  );
};

export default Header;
