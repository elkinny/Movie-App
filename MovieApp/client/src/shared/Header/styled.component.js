import styled from 'styled-components';

export const StyledHeader = styled.header`{
  padding: 20px;
  width: 100%;
  display: flex;
  align-items: center;
  justify-content: center;
`;

export const StyledTitle = styled.span`
  font-size: 40px;
  font-weight: 700;
  text-align: center;
  margin-bottom: 40px;
  text-shadow: 1px 5px 20px rgba(255, 255, 255, 0.4);
`;
