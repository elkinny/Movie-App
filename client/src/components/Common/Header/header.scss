.header{
  padding: 20px;
}

.title {
  font-size: 40px;
  text-align: center;
  margin-bottom: 40px;
  text-shadow: 1px 5px 20px rgba(255, 255, 255, .4);
}