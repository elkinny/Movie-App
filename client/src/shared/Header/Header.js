import React from 'react';
import { Link } from 'react-router-dom';

import styles from './header.scss';

const Header = () => {
  return (
    <header className={styles.header}>
      <Link to="/">
        <span className={styles.title}>Movie app</span>
      </Link>
    </header>
  );
};

export default Header;
