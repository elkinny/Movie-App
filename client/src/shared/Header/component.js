import React from 'react';

import styles from './style.scss';

const Header = () => {
  return (
    <header className={styles.header}>
      <span className={styles.title}>Movie app</span>
    </header>
  );
};

export default Header;
