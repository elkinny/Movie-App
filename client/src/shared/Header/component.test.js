import Header from './Header';
import React from 'react';
import { mount } from 'enzyme';
import { BrowserRouter } from 'react-router-dom';

describe('Header: ', () => {
  it('Header to match snapshot', () => {
    const component = mount(
      <BrowserRouter>
        <Header />
      </BrowserRouter>,
    );
    expect(component).toMatchSnapshot();
  });
});
