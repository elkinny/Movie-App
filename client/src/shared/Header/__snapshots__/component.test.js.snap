// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header:  Header to match snapshot 1`] = `
<BrowserRouter>
  <Router
    history={
      Object {
        "action": "POP",
        "block": [Function],
        "createHref": [Function],
        "go": [Function],
        "goBack": [Function],
        "goForward": [Function],
        "length": 1,
        "listen": [Function],
        "location": Object {
          "hash": "",
          "pathname": "/",
          "search": "",
          "state": undefined,
        },
        "push": [Function],
        "replace": [Function],
      }
    }
  >
    <Header>
      <header
        className="header"
      >
        <span
          className="title"
        >
          Movie app
        </span>
      </header>
    </Header>
  </Router>
</BrowserRouter>
`;
